CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+The documentation for each repository is available in the "**docs/**" directory of that repository.
+It is written in [Markdown format] and built using [MkDocs].
+
+## Testing Your Changes Locally
+
+We **strongly** encourage you to test your changes locally before contributing them.
+To do so, you first need to do three things; these are:
+
+1. Have MkDocs installed.
+2. Have the Laminas documentation theme available to the laminas-form directory.
+3. Update `mkdocs.yml` to build the docs using the documentation theme.
+
+To complete these steps, please follow the instructions below.
+
+1. [Install MkDocs].
+2. Clone the [documentation-theme repository] locally.
+    ```
+    git clone git@github.com:laminas/documentation-theme.git documentation-theme
+    ```
+3. Copy the theme directory from the cloned documentation-theme repository to the root directory of this project.
+4. Replace lines 91 and 92 of `mkdocs.yml` with the following Yaml snippet, changing them to suit the repository that you are working with.
+    ```yaml
+    # The following example assumes that you are working with laminas-form
+    repo_url: 'https://github.com/laminas/laminas-form'
+    extra:
+        repo_name: laminas/laminas-form
+        project: Components
+        base_url: https://docs.laminas.dev/
+        project_url: https://docs.laminas.dev/components/
+    theme:
+        name: null
+        custom_dir: 'theme/'
+        static_templates:
+          - pages/404.html
+    markdown_extensions:
+        - pymdownx.superfences:
+        - pymdownx.tabbed:
+        - toc:
+            toc_depth: 2
+    ```
+
+When you have completed each of these steps then, from the root directory of the project, run `mkdocs serve &`.
```

Comment:
Is the ampersand needed? On Windows this causes an error. It compiles however with `mkdocs serve`.

Reply:
No.

Good catch! 👍